### ISSUE TITLE
Error: invalid message: must have author

### ISSUE THREAD

**Issue by christianbundy** [ssbc/ssb-validate#13]
It looks like bench-ssb has stopped working locally for me because of an error being thrown by ssb-validate@4.0.0. Here's a minimal test case:

```js
V = require('ssb-validate')

var state = V.initial()
var msg = { key: '%DG5Qxcu5JFDfHoo0ffBgcgzsZHwpVn3uj6LfUhVQvLI=.sha256',
  value: 
   { previous: null,
     sequence: 1,
     author: '@XJzsDayjP5MmCU11q5Nfy6PZArMDMAegddjpB1ECw2k=.ed25519',
     timestamp: 1541175561248,
     hash: 'sha256',
     content: 
      { type: 'contact',
        contact: '@c/Azh+gM2d4unMDw1MQPzFn3yUbPiL1DDF2WCsEu/1E=.ed25519',
        following: true },
     signature: '9U6msEtGpdTb09mNWZOw3r0lsB8Rq+vrk2kLYDz83j4xFm4IgXuQVHo1q5IxYtXr1x267pt2YQM6OUN5j66KBg==.sig.ed25519' },
  timestamp: 1541175561250 }

V.append(state, null, msg)
```

```
Error: invalid message: must have author
    at Object.exports.checkInvalidCheap (/home/christianbundy/src/ssbc/bench-ssb/node_modules/ssb-validate/index.js:110:12)
    at Object.exports.checkInvalid (/home/christianbundy/src/ssbc/bench-ssb/node_modules/ssb-validate/index.js:142:21)
    at Object.exports.append_kvt (/home/christianbundy/src/ssbc/bench-ssb/node_modules/ssb-validate/index.js:200:20)
    at Object.exports.append (/home/christianbundy/src/ssbc/bench-ssb/node_modules/ssb-validate/index.js:229:18)
```

**Comment by christianbundy** [comment]
cc: @arj03

**Comment by dominictarr** [comment]
fixed in 14f65947261adb964ac616f9295a0e62b4950fea

**Comment by christianbundy** [comment]
Sweet, thanks for the quick fix!
